.goreleaser.yaml

```diff
@@ -7,6 +7,20 @@ builds:
       - windows
       - darwin
     binary: tagger
+dockers:
+  # build latest and specific tag version images
+  - image_templates:
+      - "linode/{{.ProjectName}}:{{ .Tag }}"
+      - "linode/{{.ProjectName}}:latest"
+      # push to github container registry too
+      - "ghcr.io/linode/{{.ProjectName}}:{{ .Tag }}"
+      - "ghcr.io/linode/{{.ProjectName}}:latest"
+    goos: linux
+    goarch: amd64
+    use: docker
+    # goreleaser needs to specify the config file for Dockerfile COPY
+    extra_files:
+      - ./packaging/etc/tagger.yml
```

Comment:
```suggestion
```

I'd favor bind mounting in a config, rather than baking one in. Whatever config file we bake into the image won't be enough for all user's use cases and would need to be overwritten to really be "useful" anyway.